Dockerfile

```diff
@@ -72,6 +72,12 @@ RUN mix assets.deploy
 # Compile the release
 RUN mix compile
 
+# Clean up unnecessary files after compilation to reduce image size
+RUN rm -rf content \
```

Comment:
Heads up, since this is only for building and the actual release layer is started down below `FROM ${RUNNER_IMAGE}`, this only changes the layer cache size. Not the final image size.

Reply:
Any suggestions you have are welcome. Doesn’t work as is, led to a 500 on fly